app.go

```diff
@@ -562,7 +562,7 @@ func New(opts ...Option) *App {
 	app.provide(provide{Target: app.dotGraph, Stack: frames})
 
 	if app.err != nil {
-		app.log.LogEvent(&fxevent.ProvideError{Err: app.err})
+		app.log.LogEvent(&fxevent.Provide{Err: app.err})
```

Comment:
we should probably do this in the `app.provide`, including both, the constructor that we failed to provide and the error we got back in one event. (So we just need to update that `defer`.)

Reply:
Offline discussion suggested that this PR is limited to renames only, and not changing which events get emitted when.